raiden/transfer/node.py

```diff
@@ -845,6 +848,20 @@ def handle_receive_transfer_refund_cancel_route(
     return subdispatch_to_paymenttask(chain_state, state_change, new_secrethash, storage)
 
 
+def handle_receive_transfer_refund_cancel_route_light(
+    chain_state: ChainState, state_change: ActionTransferRerouteLight, storage
+) -> TransitionResult[ChainState]:
+
+    new_secrethash = state_change.secrethash
+    current_payment_task = chain_state.payment_mapping.secrethashes_to_task[
```

Comment:
This is already made at https://github.com/rsksmart/lumino/blob/develop/raiden/api/python.py#L1831

Basically, this is the way of create a new payment task associated with the same payment 

This isnt correct because we are doing it twice

Reply:
Im lost here again, this is a replicate of the code above:

```
def handle_receive_transfer_refund_cancel_route(
    chain_state: ChainState, state_change: ActionTransferReroute, storage
) -> TransitionResult[ChainState]:

    new_secrethash = state_change.secrethash
    current_payment_task = chain_state.payment_mapping.secrethashes_to_task[
        state_change.transfer.lock.secrethash
    ]
    chain_state.payment_mapping.secrethashes_to_task.update(
        {new_secrethash: copy.deepcopy(current_payment_task)}
    )
    return subdispatch_to_paymenttask(chain_state, state_change, new_secrethash, storage)
```
I just added that to support the new ActionTransferRerouteLight event that's all. Maybe this is related to the change of the secret hash that you mentioned to me.

Reply:
Accordly to what we have talked, we should remove the clone and just return the subdistpatch